source/MRTestC/MRMeshTopology.c

```diff
@@ -0,0 +1,103 @@
+#include "TestMacros.h"
+#include <MRMeshC/MRMeshTopology.h>
+#include <MRMeshC/MRCube.h>
+#include <MRMeshC/MRMesh.h>
+#include <stdio.h>
+
+
+void testMrMeshTopologyPack(void)
+{
+    MRVector3f size = mrVector3fDiagonal(1.0f);
+    MRVector3f base = mrVector3fDiagonal(-0.5f);
+    MRMesh* mesh = mrMakeCube(&size, &base);
```

Comment:
You might want to move the mesh creation to a separate function as it is used multiple times in this file.
```cpp
MRMesh* makeCube( void )
{
    MRVector3f size = mrVector3fDiagonal( 1.0f );
    MRVector3f base = mrVector3fDiagonal( -0.5f );
    return mrMakeCube( &size, &base );
}
...
MRMesh* mesh = makeCube();
```